src/Autodesk.Forge.Core/ForgeConfiguration.cs

```diff
@@ -29,5 +29,11 @@ public ForgeConfiguration()
         public string ClientId { get; set; }
         public string ClientSecret { get; set; }
         public Uri AuthenticationAddress { get; set; }
+
+        /// <summary>
+        /// Timeout in seconds.
+        /// By default - 10 seconds.
+        /// </summary>
+        public int Timeout { get; set; } = 10;
```

Comment:
The forge gateway imposes a 10s timeout on all requests. So it makes little sense to configure anything longer than 10s when calling Forge. If the resiliency policy is used in another context (as in case of Das SDK when it uploads an AppBundle to S3 using this timeout) then of course longer timeouts make sense but we should configure it per-call rather than globally like this. I recommend that we improve the code to honor a timeout that is set via `HttpRequestMessage.Properties`. Something like `
"ForgeTimeout"=seconds. Then the Das SDK could set this key when it is doing an upload to s3.

Reply:
@szilvaa Please, check if approach is correct. I'll add unit tests if it is

Reply:
This is the PR for Das SDK, where the timeout is used: https://github.com/Autodesk-Forge/forge-api-dotnet-design.automation/pull/2